spec/support/matchers.rb

```diff
@@ -39,7 +39,9 @@
     doc = actual
 
     doc.css("a[href^='/'],a[href^='#{SITE_URL}']").each do |a|
-      next if a[:lang]
+      # `hreflang` implies that it's intentional for the link target language to differ from the
+      # current page language.
```

Comment:
Splitting hairs a bit, but does this mean we should not include `hreflang` for the English link if we're currently viewing in English?

Reply:
I agree it's redundant, but it's harmless to be present, and avoids complexity of maintaining logic to exempt the current locale.